<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reportmanager</class>
 <widget class="QDialog" name="reportmanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>393</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>طباعة</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>444</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dev butt</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>1</y>
     <width>16</width>
     <height>479</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="report_type">
   <property name="geometry">
    <rect>
     <x>456</x>
     <y>156</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>تقرير مشتريات</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>تقرير مبيعات</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>جرد اصناف</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ارباح و الخسائر</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="start">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>22</y>
     <width>116</width>
     <height>23</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>568</x>
     <y>19</y>
     <width>50</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>من</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="end">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>64</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>62</y>
     <width>50</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>الي</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>105</y>
     <width>50</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>المخزن</string>
   </property>
  </widget>
  <widget class="QComboBox" name="storage">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>107</y>
     <width>112</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>6</y>
     <width>403</width>
     <height>472</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="buy_page">
    <widget class="QCheckBox" name="pur_purchases_check">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>15</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>مشتريات</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>58</y>
       <width>287</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="pur_supp_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>المورد</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pur_sup_name">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="pur_return_check">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>16</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>مرتجع مشتريات</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>132</y>
       <width>81</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="pur_barcode_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>باركود</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pur_int_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>كود دولي</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pur_item_name_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>اسم الصنف</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>128</y>
       <width>157</width>
       <height>154</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="pur_barcode"/>
      </item>
      <item>
       <widget class="QLineEdit" name="pur_int"/>
      </item>
      <item>
       <widget class="QComboBox" name="pur_item_name">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="sell_page">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>283</x>
       <y>178</y>
       <width>104</width>
       <height>147</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="sale_barcode_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>باركود</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sale_int_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>كود دولي</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sale_item_name_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>اسم الصنف</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>123</x>
       <y>177</y>
       <width>157</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="sale_barcode">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sale_int">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sale_item_name">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>80</y>
       <width>281</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="sale_custom_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>العميل</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sale_custom_name">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>117</y>
       <width>180</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="sale_cash_depo_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>الخزنة</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sale_cash_depo_name">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>213</x>
       <y>149</y>
       <width>175</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="sale_seller_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>البائع</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sale_seller_name">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>6</y>
       <width>189</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="sale_sales_check">
        <property name="text">
         <string>مبيعات</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sale_return_check">
        <property name="text">
         <string>مرتجع مبيعات</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>43</y>
       <width>169</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QCheckBox" name="sale_cash_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>نقدي</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sale_credit_check">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>اجل</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="inven_page">
    <widget class="QComboBox" name="inv_type">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>19</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>تقرير  حالي</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>تقرير فترة</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>60</y>
       <width>81</width>
       <height>148</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="inv_supplier_check">
        <property name="text">
         <string>مورد</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="inv_categ_check">
        <property name="text">
         <string>قسم</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="inv_season_check">
        <property name="text">
         <string>موسم</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="inv_class_check">
        <property name="text">
         <string>تصنيف</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>154</x>
       <y>61</y>
       <width>144</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QComboBox" name="inv_supplier_name">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="category_name">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="season_name">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="class_name">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>214</y>
       <width>143</width>
       <height>107</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QComboBox" name="inv_item_name">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inv_item_int">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inv_item_bcode">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>215</y>
       <width>86</width>
       <height>98</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QRadioButton" name="inv_item_name_check">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="inv_item_int_check">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="inv_item_bcode_check">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page"/>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>472</y>
     <width>655</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>644</x>
     <y>0</y>
     <width>16</width>
     <height>481</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
